test/unit/sso/apache_test.rb

```diff
@@ -30,6 +30,9 @@ def test_available?
 
     Setting["authorize_login_delegation_api"] = false
     assert !apache.available?
+
+    Setting['HTTP_OIDC_ACCESS_TOKEN'] = 'JWT'
```

Comment:
isn't this still disabled by line 31? `Setting["authorize_login_delegation_api"]` didn't get reset before this test